doc/usecases/elektra_web/UC_duplicating_keys.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Duplicating keys
```

Comment:
Do we really need this as extra usecase? Isn't this copy&paste or ctrl+drag&drop?